src/weatherreport/src/weatherreport_check_membership.erl

```diff
@@ -0,0 +1,64 @@
+%% -------------------------------------------------------------------
+%%
+%% riaknostic - automated diagnostic tools for Riak
```

Comment:
And here

Reply:
Another original riaknostic check (note that below it refers to the "ring", a dead givaway ;)